### ISSUE TITLE
Bring back science processing

### ISSUE THREAD

**Issue by steamp0rt** [steamp0rt/Kerbalism#231]
* Bring science processing back
* Science can only be increased by 1.5x the initial science output of the experiment
* Constant electricity use
* Can cause moderate stress to the scientists in the science lab

**Comment by SirMortimer** [comment]
Ideas:

- science bonus only applied when science is researched on or around the body it was gathered on
- sample value decays over time, so better convert them to data in a lab or loose half the value on the way back to kerbin

**Comment by theJesuit** [comment]
I like SirMortimer idea of only getting the bonus with analysed in the same biome.

Don't like the decay option. A temp reading is a temp reading regardless.

Idea: only once per experiment otherwise it is an exploit.

Additional/ alternative idea: have the situational analysis experiment that IS repeatable, like the crew report and or eva report.  Have those as only ones with doubling effect but repeatable ONLY with DIFFERENT kerbals.  So to get the crew report again you need to send a second or third or fourth etc kerbal / crew or scientist only or something.

**Comment by PiezPiedPy** [comment]
@theJesuit As far as I know, the decay was for samples only and not data

**Comment by HaullyGames** [comment]
I like the decay idea, this force the player do something with that sample.

**Comment by Gordon-Dry** [comment]
Depends the sample; is it green and slimy or is it just dust and gravel?

**Comment by theJesuit** [comment]
Not defined @Gordon-Dry that is why it has to go to the R&D chicks for testing.

**Comment by SirMortimer** [comment]
> Depends the sample; is it green and slimy or is it just dust and gravel?

How about samples from planets with an atmosphere? Bacteria could perish in space. Even dead dust does decay, that's why Hayabusa creates a new artificial crater to collect dust samples from. They want fresh soil that had not been exposed to radiation for millions of years. Of course, compared to that an interplanetary trip is nothing and probably won't do anything to a dust sample - especially when it's transported in a container that provides a minimum amount of shielding.

But that's not the point. It would be an incentive to use science labs.

**Comment by SirMortimer** [comment]
Another shower thought on this:

Dont't allow samples to be stored in cores, leave them in the experiments that generated them, like stock KSP does. Bring back the science container as one of very few parts that actually can store a limited amount of samples. Transferral of a sample from the experiment to anywhere should require a Kerbal, I don't see how a blob of glibbery green goo moves from one place to another without any physical interaction.

Limiting the amount of storable samples would be an incentive to use science labs to process them into transmittable data. 

And while at it, also limit the amount of science data that can be stored, because why not?

**Comment by steamp0rt** [comment]
obsolete with 3.0, pretty much
